src/views/Player.vue

```diff
@@ -148,6 +155,8 @@ export default defineComponent({
       const questionSet = quizDetails.question_sets[0];
       state.questions = questionSet.questions;
       state.metadata = quizDetails.metadata;
+      state.maxMarks =
+        quizDetails.max_marks || quizDetails.num_graded_questions;
```

Comment:
If `max_marks` is not available, why would `num_graded_questions` be a correct alternative for maxMarks?